### DISCUSSION TITLE
[양태민] 과제 1번 제출합니다.

### DISCUSSION THREAD

**Discussion by ytm97** [SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course#28] [2024년도 과제 제출]
![image](https://github.com/SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course/assets/172099563/a2b1fc05-ff16-421e-ba53-d38f51f11b4e)

**Comment by Jinsun-Lee** [comment]
안녕하세요.
자율주행 통합실습 담당 조교입니다. 

아래 링크에서 교육 수강 전 공지사항을 확인해 주시길 바랍니다.
https://github.com/SKKUAutoLab/H-Mobility-Autonomous-Advanced-Course/discussions/15

감사합니다.
